lib/crystalball/source_diff/file_diff.rb

```diff
@@ -37,6 +37,21 @@ def new_relative_path
         git_diff.patch.match(/rename from.*\nrename to (.*)/)[1]
       end
 
+      def whitespace?
```

Comment:
I think we should add different method that will return "trimmed" diff and check if it is empty